zerver/tests/test_alert_words.py

```diff
@@ -13,7 +13,9 @@ class AlertWordTests(ZulipTestCase):
     def get_user(self) -> UserProfile:
         # One nice thing about Hamlet is that he is
         # already subscribed to Denmark.
-        return self.example_user('hamlet')
+        user = self.example_user('hamlet')
+        AlertWord.objects.filter(user_profile=user).delete()
```

Comment:
Let's comment this test setup line noting that `populate_db` makes some.

Reply:
fixed